Proxy/Configuration/Response/Json/index.ts

```diff
@@ -130,6 +130,21 @@ export namespace Json {
 			)
 		}, body)
 	}
+	export function reverseProcess(
+		configuration: Json,
+		variables: Record<string, any>,
+		body: Record<string, any>
+	): Record<string, any> {
```

Comment:
can you add docs to this please? ideally also the rest of the stuff in this class would be great
https://tsdoc.org/ same format as javadoc